opm/simulators/linalg/FlexibleSolver_impl.hpp

```diff
@@ -180,47 +188,54 @@ namespace Dune
+                                                                                            tol,// desired residual reduction factor
+                                                                                            restart,
+                                                                                            maxiter, // maximum number of iterations
+                                                                                            verbosity);
 #if HAVE_SUITESPARSE_UMFPACK
-        } else if (solver_type == "umfpack") {
-            if constexpr (std::is_same_v<typename VectorType::field_type,float>) {
-                OPM_THROW(std::invalid_argument, "UMFPack cannot be used with floats");
-            } else {
-            using MatrixType = std::remove_const_t<std::remove_reference_t<decltype(linearoperator_for_solver_->getmat())>>;
-            linsolver_ = std::make_shared<Dune::UMFPack<MatrixType>>(linearoperator_for_solver_->getmat(), verbosity, false);
-            direct_solver_ = true;
-            }
+                } else if (solver_type == "umfpack") {
+                    if constexpr (std::is_same_v<typename VectorType::field_type,float>) {
+                        OPM_THROW(std::invalid_argument, "UMFPack cannot be used with floats");
+                    } else {
+                        using MatrixType = std::remove_const_t<std::remove_reference_t<decltype(linearoperator_for_solver_->getmat())>>;
+                        linsolver_ = std::make_shared<Dune::UMFPack<MatrixType>>(linearoperator_for_solver_->getmat(), verbosity, false);
+                        direct_solver_ = true;
+                    }
 #endif
 #if HAVE_CUDA
-        } else if (solver_type == "gpubicgstab") {
-            linsolver_.reset(new Opm::gpuistl::SolverAdapter<Operator, Dune::BiCGSTABSolver, VectorType>(
-                *linearoperator_for_solver_,
-                *scalarproduct_,
-                preconditioner_,
-                tol, // desired residual reduction factor
-                maxiter, // maximum number of iterations
-                verbosity,
-                comm));
-#endif
-        } else {
+                } else if (solver_type == "gpubicgstab") {
+                    linsolver_.reset(new Opm::gpuistl::SolverAdapter<Operator, Dune::BiCGSTABSolver, VectorType>(
+                        *linearoperator_for_solver_,
+                        *scalarproduct_,
+                        preconditioner_,
+                        tol, // desired residual reduction factor
+                        maxiter, // maximum number of iterations
+                        verbosity,
+                        comm));
+                }
+            }
+        }
+        #endif
+        if (!linsolver_) {
             OPM_THROW(std::invalid_argument,
-                      "Properties: Solver " + solver_type + " not known.");
+                    "Properties: Solver " + solver_type + " not known.");
```

Comment:
Looks like your editor or setup makes strange indentation changes for no reason?